api/logs/views.py

```diff
@@ -220,7 +221,7 @@ class NodeLogContributors(JSONAPIBaseView, generics.ListAPIView, ODMFilterMixin,
     required_read_scopes = [CoreScopes.USERS_READ]
     required_write_scopes = [CoreScopes.NULL]
 
-    serializer_class = UserSerializer
+    serializer_class = UnregisteredContributorSerializer
```

Comment:
My only concern is that user information not be serialized when there's an anonymous link. This seems fine, but please write some tests so if someone decides that they need log params when there's an anonymous link, that user info is not revealed,